contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -92,6 +93,53 @@ contract LidoOracle is ILidoOracle, AragonApp {
     bytes32 internal constant LAST_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.lastCompletedEpochId");
     bytes32 internal constant PREV_COMPLETED_EPOCH_ID_POSITION = keccak256("lido.LidoOracle.prevCompletedEpochId");
 
+    /*
+    If we use APR as a basic reference for increase, and expected range is between 1 and 10% APR.
+    So the granularity step for yearly APR is 1%
+    daily_granularity = 0.01 / 365 = 2.74e-05
+    Currently used basisPoints (1e-4) is unable to express such microscopic ratio.
+    So we should take another popular metric to express this - PPM or 1e-6
+    daily_granularity_in_ppm = 2.74e-05 / 1e-6 = 27 (pretty good)
+    with 100% buffer for future spikes it will be 50.
+    May be changed by Gov.
+    */
+    bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE_POSITION =
+        keccak256("lido.LidoOracle.allowedBeaconBalanceAnnualRelativeIncrease");
+    uint256 internal constant DEFAULT_ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_INCREASE = 50;  // PPM
+
+    /*
+    When slashing happens, the balance may decrease at a much faster pace.
+    On the start the system will have about 100 validators and we can guess one of them can be slashed for any reason.
+    1/100 = 10000 PPM.
+    May be changed by Gov.
+    */
+    bytes32 internal constant ALLOWED_BEACON_BALANCE_ANNUAL_RELATIVE_DECREASE_POSITION =
```

Comment:
it's handled per report but the jump is annualized

Reply:
@vshvsh, do you mean that this threshold should be daily instead of annually? 
("Daily" = "per-report" normally, because our reports happen once a frame and the frame is one day long).

Reply:
Not daily. A maximum decrease since the last report, whenever it was, is limited to a fixed relative decrease (e.g. 15%). You had it right in the previous implementation.

Reply:
To explain better:
There can be a very rare, but a sane one-time decrease of total balance by ~5%. 
If we annualize that, we get an unreasonable annualized 1500%-is boundary, which is not sane. Even 10% one-time decrease in two days (e.g. if the oracle is down for two days) is not sane. We need to allow a single slashing to be ~5-10%, not all slasings collectively to be 1500% per year.

Reply:
It looks like I should change the mechanism.
What I can do is that I can
1) store one report per week/month and calculate relative jump for a significant amount of time not just between two reports.
2) calculate something like Smoothed EWMA(Balance) and check the jump `currentBalance - EWMA(Balance)`, this way if there will be several consecutive big jumps it's sane (EWMA track the balance smoothly), but if it's rare big jumps then it's OK

Reply:
discussed with @vshvsh 
decided just to leave one jump check (not annualized)
done